### DISCUSSION TITLE
Reset trigger in ERROR state?  Java has scheduler.resetTriggerFromErrorState

### DISCUSSION THREAD

**Discussion by jafin** [quartznet/quartznet#1892] [Q&A] [answered]
Just searching for a way to reset a trigger in `ERROR` state to resume it.  Quartz.net has `scheduler.ResumeTrigger`, but for `PAUSED` state.
I see the java lib has `resetTriggerFromErrorState`.    [resetTriggerFromErrorState](https://www.tabnine.com/code/java/methods/org.quartz.spi.JobStore/resetTriggerFromErrorState)(triggerKey)

Does Quartz.Net have any way to perform a trigger reset from ERROR? (I couldn't find one)
If not, would a PR to add `resetTriggerFromErrorState(TriggerKey) be useful?

Thanks.

**Comment by lahma** [comment, marked as answer]
I don't know how I've missed this when going through Java side's commits, but definitely something to port over! [This seems to be the PR which was merged](https://github.com/quartz-scheduler/quartz/pull/125).

So a PR would be much appreciated, targeting the 3.x branch, thank you in advance.

**Comment by lahma** [comment]
I went ahead and ported the feature here https://github.com/quartznet/quartznet/pull/1904 , I'll release 3.6 soon and it will be part of it.
